src/Form/Layout/Section/Accordion.php

```diff
@@ -30,7 +30,7 @@ protected function init(): void
             $jsError = [$form->js()->form('add prompt', $fieldName, $str)];
 
             // if a form control is part of an accordion section, it will open that section.
-            $section = $form->getClosestOwner($form->getControl($fieldName), AccordionSection::class);
+            $section = $form->getControl($fieldName)->getClosestOwner(AccordionSection::class);
```

Comment:
@mkrecek234 the `getClosestOwner` method was broken, does this fix https://github.com/atk4/ui/issues/1728 too?

Reply:
@mkrecek234 should be fixed when not nested